tests/test_scripts.py

```diff
@@ -24,6 +24,14 @@ def test_setup_command_line_enableDetective():
 
     args = enableDetective.setup_command_line(['--master_account', '000000000001', '--assume_role', 'detectiveAdmin', '--enabled_regions', 'us-east-1,us-east-2,us-west-2,ap-northeast-1,eu-west-1', '--input_file', 'accounts.csv'])
     assert args.master_account == '000000000001'
+    assert args.tags == None
+
+    args = enableDetective.setup_command_line("--master_account 123456789012 --assume_role detectiveAdmin --input_file accounts.csv --tags TagKey1=TagValue1,TagKey2=TagValue2,TagKey3=TagValue3".split(" "))
+    assert args.tags == {
+        "TagKey1": "TagValue1",
+        "TagKey2": "TagValue2",
+        "TagKey3": "TagValue3",
```

Comment:
Can you test key with no value set?

Also, do we test negative cases here? Might be good to test a case of invalid tags (no key, key/value with invalid characters, 51+ tags)

Reply:
This test is primarily focused on parsing the command line arguments for the script. To that end, I'll adding a case for a tag key with no associated value.

Adding the other test cases that you mentioned doesn't make as much sense to me, since those constrains are enforced by the service and also checked by the boto3 client.

To add those cases, we would either need to duplicate the implementation and enforcement of those constrains here, or effectively transform these tests into integration tests, to verify that the boto3 client and the Detective service are enforcing the constrains that they claim to posses. I do not see much value in either option.

Reply:
looks good!